src/main/app/idam/serviceAuthToken.ts

```diff
@@ -1,5 +1,13 @@
+import * as moment from 'moment'
+import { JwtUtils } from 'common/utils/jwtUtils'
+
 export default class ServiceAuthToken {
   constructor (public bearerToken: string) {
     this.bearerToken = bearerToken
   }
+
+  hasExpired (): boolean {
+    const { exp } = JwtUtils.decodePayload(this.bearerToken)
+    return moment().unix() > exp
```

Comment:
when `exp` === moment().unix() is it not expired?

Reply:
Good point, JWT spec says:

> The "exp" (expiration time) claim identifies the expiration time on
   or after which the JWT MUST NOT be accepted for processing.  The
   processing of the "exp" claim requires that the current date/time
   MUST be before the expiration date/time listed in the "exp" claim.